config/index.js

```diff
@@ -1,5 +1,6 @@
 exports.APP = {
-  website: 'https://ark.io'
+  website: 'https://ark.io',
+  transactionExpiryMinutes: 45
```

Comment:
Maybe this can be set as a network property, so, when core provides the configuration, the transition would be straight